### PR TITLE
Fixed an error in the definition of a python process.

### PR DESCRIPTION
**Pull request by ipouplana**
I also added a comment in the readme.txt.

### CODE REVIEW

File: applications/DamApplication/README.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@ Note: For the moment, MPI only works in Linux and requires compiling METIS_APPLI
 -DCONVECTION_DIFFUSION_APPLICATION=ON 						                              \
 -DEXTERNAL_SOLVERS_APPLICATION=ON						                                    \
 -DSOLID_MECHANICS_APPLICATION=ON   					                                    \
+-DFLUID_DYNAMICS_APPLICATION=ON   					                                    \
```
Comment (lorenzogracia) [inline applications/DamApplication/README.txt]: I think this is not necessary for Dam Application

**Review by lorenzogracia — CHANGES_REQUESTED** [review]

**Review by lorenzogracia — APPROVED** [review]

**Comment by ipouplana** [comment]
Hi @roigcarlo, Travis checks have failed because: "The job exceeded the maximum time limit for jobs, and has been terminated" and this blocks the possibility of merging this PR, which has no changes in any C++ file (only python and txt).
So, I was wondering, is there any way to distinguish PRs that introduce changes in C++ files and need to be compiled before merging from those that do not need to be compiled (like this one)? That could reduce the cues in the Travis checks...

**Comment by roigcarlo** [comment]
Sadly travis does not know if we have touched a cpp or a python. But there are ways to apply filters. I want to discuss this in the next meeting

**Comment by ipouplana** [comment]
Perfect, thanks @roigcarlo !

**Comment by ipouplana** [comment]
@roigcarlo sorry to disturb you with this PR, but the Trevis checks have failed again with the same message: "The job exceeded the maximum time limit for jobs, and has been terminated"...
Does the Travis check restart automatically? If not, how can I restart it?
Thanks

**Comment by roigcarlo** [comment]
There is a button in the travis page but you must log in with the github
account, let me re-restart it

**Comment by roigcarlo** [comment]
Now it passes :+1:

**Pull request merged**